db/seeds.rb

```diff
@@ -6,8 +6,17 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Lifehack.create(
-  [{ title: 'How to tie a tie',
-    description: 'tie a knot' },
-    { title: 'How to tie a tie1' }]
+User.create(
+  [{ first_name: 'Jon',
+    last_name: 'Knot',
+    email: 'jon@gmail.com',
+    password: 'whattheheck?' }]
 )
+
+  Lifehack.create(
+    [{ title: 'How to tie a tie',
+      description: 'tie a knot',
+      creator_id: "#{User.first.id}"},{
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space missing after comma.<br>Space inside } missing.